qualtran/bloqs/factoring/ecc/ec_phase_estimate_r.py

```diff
@@ -70,11 +70,8 @@ def build_composite_bloq(self, bb: 'BloqBuilder', x: Soquet, y: Soquet) -> Dict[
     def build_call_graph(self, ssa: 'SympySymbolAllocator') -> Set['BloqCountT']:
         return {(ECAddR(n=self.n, R=self.point), self.n), (MeasureQFT(n=self.n), 1)}
 
-    def pretty_name(self) -> str:
-        point_str = str(self.point)
-        if len(point_str) < 5 or True:
-            return f'PE${point_str}$'
-        return 'PE(R)'
+    def __str__(self) -> str:
```

Comment:
Any concerns about dropping the long string condition?

Reply:
if you look closely, this seems to actually have been some debug code that snuck through (mea culpa) because of the `or True` condition.